src/api.js

```diff
@@ -79,21 +89,27 @@ export default class Api {
 
     return fetch(recordsUrl + queryString, {headers})
       .then(res => {
+        response = res;
```

Comment:
You mean using res directly? We're assigning the result to the response var available from the outter scope, so next Promise steps can still access the response object.